src/sentry/grouping/strategies/message.py

```diff
@@ -1,9 +1,82 @@
 from __future__ import absolute_import
 
+import re
+import six
+from itertools import islice
+
 from sentry.grouping.component import GroupingComponent
 from sentry.grouping.strategies.base import strategy
 
 
+_irrelevant_re = re.compile(r'''(?x)
+    (?P<email>
+        [a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*
+    ) |
+    (?P<url>
+        \b(wss?|https?|ftp)://[^\s/$.?#].[^\s]*
+    ) |
+    (?P<uuid>
+        \b
+            [0-9a-fA-F]{8}-
+            [0-9a-fA-F]{4}-
+            [0-9a-fA-F]{4}-
+            [0-9a-fA-F]{4}-
+            [0-9a-fA-F]{12}
+        \b
+    ) |
+    (?P<sha1>
+        \b[0-9a-fA-F]{40}\b
+    ) |
+    (?P<md5>
+        \b[0-9a-fA-F]{32}\b
+    ) |
```

Comment:
Should IPv4 patterns be included in this noisy data list?

Reply:
Indeed, that sounds like a good idea.